src/validation.h

```diff
@@ -187,34 +188,65 @@ void PruneBlockFilesManual(CChainState& active_chainstate, int nManualPruneHeigh
 * Validation result for a single transaction mempool acceptance.
 */
 struct MempoolAcceptResult {
-    /** Used to indicate the results of mempool validation,
-    * including the possibility of unfinished validation.
-    */
+    /** Used to indicate the results of mempool validation. */
     enum class ResultType {
```

Comment:
nit: can inherit from u8 to make smaller

Reply:
make sure to sort the m_result_type after m_state for packing rules.

Reply:
Most compilers will word-align it, no?

Reply:
unclear to me what exact size TxValidationState is.

General rule of thumb unless you know something about accesses occurring together often is to sort in order biggest field to smallest, it guarantees good packing IIRC.